apps/ejabberd/priv/cassandra.cql

```diff
@@ -1,4 +1,77 @@
-DROP KEYSPACE IF EXISTS mongooseim;
 CREATE KEYSPACE mongooseim
          WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': 3};
 USE mongooseim;
+
+CREATE TABLE mam_message(
```

Comment:
I guess it is. Just after I finish fixing up code-style related issues, I'll look into it to be sure and I'll let you know.